components/organisms/Repositories/repositories.tsx

```diff
@@ -146,7 +151,7 @@ export default function Repositories({ repositories, showSearch = true }: Reposi
             <div className="flex justify-between p-3 px-6 items-center border-b-2 text-light-slate-11">
               <div>{selectedRepos.length} Repositories selected</div>
               <Button onClick={handleOnAddtoInsights} variant="primary">
-                Add to Insight Page
+                {workspaceId ? "Add to Insight Page" : "Add to Workspace"}
               </Button>
             </div>
```

Comment:
Since this is being updated for workspaces, might as well make it more accessible as well.

```suggestion
          <div
            aria-hidden={selectedRepos.length === 0}
            className={clsx(
              selectedRepos.length > 0 ? "flex" : "hidden",
              `justify-between p-3 px-6 items-center border-b-2 text-light-slate-11`
            )}
          >
            <div>{selectedRepos.length} Repositories selected</div>
            <Button onClick={handleOnAddtoInsights} variant="primary">
              {workspaceId ? "Add to Insight Page" : "Add to Workspace"}
            </Button>
          </div>
```